### DISCUSSION TITLE
Modelling shaking table experiment_unexpected prescribed velocity

### DISCUSSION THREAD

**Discussion by 19Dario88** [Anura3D/Anura3D_OpenSource#220] [Q&A] [answered]
Dear all,
I’m Dario, a PhD student in Geosciences. I’m interesting in the ANURA3D applicability on seismic-induced landslides, so I started to replicate what Alsardi&Yerro (2021) presented on that topic: modelling the shaking table experiment by Hiraoka (2013): a soil body settled on a rigid horizontal plate, while a vertical wall retains the soil on the left side, trough a smooth contact (Fig.a). 
However, I’m facing with a “preliminary” numerical issue: how the nodal prescribed velocity (from external file, Fig.b) is modelled in the nodes of the horizontal plate and the vertical wall (both modelled as rigid bodies), where the prescribed velocity are implemented. As rigid bodies, I expect that in every node of the plate and of the wall, in every time step, the velocity must be the same of what I prescribed as boundary condition. However, it doesn’t happen in any node (Fig.c). Where Am I wrong?
Thank you in advance for your suggestions!
Dario
[Hiraoka2013_v16.GOM.txt](https://github.com/Anura3D/Anura3D_OpenSource/files/15179812/Hiraoka2013_v16.GOM.txt)
[Hiraoka2013_v16.CPS_001.txt](https://github.com/Anura3D/Anura3D_OpenSource/files/15179813/Hiraoka2013_v16.CPS_001.txt)
[Hiraoka2013_v16.CPS_002.txt](https://github.com/Anura3D/Anura3D_OpenSource/files/15179811/Hiraoka2013_v16.CPS_002.txt)

a)![immagine](https://github.com/Anura3D/Anura3D_OpenSource/assets/168671577/0cb20fb8-b8bb-474a-80b9-83ecb7fece07)
b)![immagine](https://github.com/Anura3D/Anura3D_OpenSource/assets/168671577/cb2217e9-daa2-4477-9e24-dd29ebd179da)
c)![immagine](https://github.com/Anura3D/Anura3D_OpenSource/assets/168671577/9283afb7-bcb1-4263-ac0e-a0d67aa6f4cf)

**Comment by 19Dario88** [comment, marked as answer]
Dear all,
after talking about this issue with other researchers, the problem has been solved. As the second column of the ".PVF" file contains multiplier coefficients of the initial prescribed velocity, the latter have to be set equal to 1 in the GID model. 
Regards,
Dario

**Reply by kuailebozai** [reply]
Hi Dario, I'm interested in your question about earthquake induced landslides because I've encountered a similar problem before with Anura3d, and if it's possible I'd like to ask you about the simulation of earthquakes. Here is my email: kaiboyang2023@163.com, please contact me, thank you very much!

**Reply by 19Dario88** [reply]
Hi,
I'm pretty new on ANURA3D, but I'm sure that users more expert than me could help you. If you need explanations about this specific issue, please ask!
Regards,
Dario

**Reply by kuailebozai** [reply]
Hi Dario, I am also trying to simulate the shaker experiment of Hiraoka (2013), however, when I do the calculations using rigid bodies and "contact", the program reports the error "forrtl: severe (157): Program Exception - access violation", so I would like to know how you solved this problem, could you provide me with your ".GID" file and your ".PVF" velocity file, it would be very helpful to me. This will help me a lot, thank you very much!

**Reply by CPLA81192** [reply]
> Dear all, after talking about this issue with other researchers, the problem has been solved. As the second column of the ".PVF" file contains multiplier coefficients of the initial prescribed velocity, the latter have to be set equal to 1 in the GID model. Regards, Dario

Hello, I am having trouble with a similar problem, and I was wondering if I could ask you for some guidance on it?
